test/prism/PrismLogicTest.scala

```diff
@@ -145,40 +149,48 @@ class PrismLogicTest extends FreeSpec with Matchers {
       val i2 = instanceWithSSA("i2", ssa2)
 
       "an SSA with an AMI" in {
-        amiSSAs(List(a1 -> List(i1))) shouldEqual Map(ssa1 -> List(a1))
+        val ami = a1.copy(instances = Some(List(i1)))
+        allSSAs(List(ami)) shouldEqual Map(ssa1 -> List(ami))
```

Comment:
The containing block's text label no longer matches the name of the function you're testing. (Currently on line 141)